edx_exams/apps/core/tests/test_api.py

```diff
@@ -212,7 +213,7 @@ def setUp(self):
             resource_id=str(uuid.uuid4()),
             course_id=self.course_id,
             provider=self.test_provider,
-            content_id='abcd1234',
+            content_id='block-v1:edX+test+2023+type@sequential+block@1111111111',
```

Comment:
This is necessary because the `content_id` must be able to be parsed into a valid `UsageKey`.